test/provision/manifest/dns_deployment.yaml

```diff
@@ -58,78 +43,58 @@ metadata:
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: kube-dns
-  namespace: kube-system
+  generation: 1
   labels:
     k8s-app: kube-dns
-    kubernetes.io/cluster-service: "true"
-    addonmanager.kubernetes.io/mode: Reconcile
+  name: kube-dns
+  namespace: kube-system
 spec:
-  # replicas: not specified here:
-  # 1. In order to make Addon Manager do not reconcile this replicas parameter.
-  # 2. Default is 1.
-  # 3. Will be tuned in real time if DNS horizontal auto-scaling is turned on.
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-app: kube-dns
   strategy:
     rollingUpdate:
       maxSurge: 10%
       maxUnavailable: 0
-  selector:
-    matchLabels:
-      k8s-app: kube-dns
+    type: RollingUpdate
   template:
     metadata:
+      creationTimestamp: null
       labels:
         k8s-app: kube-dns
-      annotations:
-        scheduler.alpha.kubernetes.io/critical-pod: ''
     spec:
-      tolerations:
-      - key: "CriticalAddonsOnly"
-        operator: "Exists"
-      volumes:
-      - name: kube-dns-config
-        configMap:
-          name: kube-dns
-          optional: true
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: beta.kubernetes.io/arch
+                operator: In
+                values:
+                - amd64
       containers:
-      - name: kubedns
-        image: k8s.gcr.io/k8s-dns-kube-dns-amd64:1.14.10
-        resources:
-          # TODO: Set memory limits when we've profiled the container for large
-          # clusters, then set request = limit to keep this container in
-          # guaranteed class. Currently, this container falls into the
-          # "burstable" category so the kubelet doesn't backoff from restarting it.
-          limits:
-            memory: 170Mi
-          requests:
-            cpu: 100m
-            memory: 70Mi
+      - args:
+        - --domain=cluster.local.
+        - --dns-port=10053
+        - --config-dir=/kube-dns-config
+        - --v=5
+        env:
+        - name: PROMETHEUS_PORT
+          value: "10055"
+        image: gcr.io/google_containers/k8s-dns-kube-dns-amd64:1.14.10
+        imagePullPolicy: IfNotPresent
```

Comment:
Hmm, this seems new. It's totally fine but I guess the old spec would use the default? (which is IfNotPresent probably)